src/tcgwars/logic/impl/gen4/CallOfLegends.groovy

```diff
@@ -1593,110 +788,80 @@ public enum CallOfLegends implements LogicCardInfo {
+            energyCost W, C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 30
+              flip self.cards.energyCount(G), {
+                damage 20
+              } 
             }
           }
 
         };
       case VULPIX_75:
-        return basic (this, hp:HP050, type:FIRE, retreatCost:1) {
-          weakness W
-          move "Singe", {
-            text "Flip a coin. If heads, the Defending Pokémon is now Burned."
-            energyCost R
-            attackRequirement {}
-            onAttack {
-              flip { apply BURNED }
-            }
-          }
-          move "Ember", {
-            text "30 damage. Energy attached to Vulpix."
-            energyCost R, C, R
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-
-        };
+        return copy(HeartgoldSoulsilver.VULPIX_87, this);
       case CHEERLEADER_S_CHEER_76:
       return copy(Unleashed.CHEERLEADER_S_CHEER_71, this);
       case COPYCAT_77:
-        return basicTrainer (this) {
-          text "You can play only one Supporter card each turn. When you play this card, put it next to your Active Pokémon. When your turn ends, discard this card.\nShuffle your hand into your deck. Then, draw a number of cards equal to the number of cards in your opponent’s hand."
-          onPlay {
-          }
-          playRequirement{
-          }
-        };
+        return copy(TeamRocketReturns.COPYCAT_83, this);
       case DUAL_BALL_78:
-        return basicTrainer (this) {
-          text "Flip 2 coins. For each heads, search your deck for a Basic Pokémon, show it to your opponent, and put it into your hand. If you do, shuffle your deck afterward."
-          onPlay {
-          }
-          playRequirement{
-          }
-        };
+        return copy(DeltaSpecies.DUAL_BALL_89, this);
       case INTERVIEWER_S_QUESTIONS_79:
-        return basicTrainer (this) {
-          text "You can play only one Supporter card each turn. When you play this card, put it next to your Active Pokémon. When your turn ends, discard this card.\nLook at the top 8 cards of your deck. Choose as many Energy cards as you like, show them to your opponent, and put them into your hand. Shuffle the other cards back into your deck."
-          onPlay {
-          }
-          playRequirement{
-          }
-        };
+        return copy(Unleashed.INTERVIEWER_S_QUESTIONS_77, this)
       case LOST_REMOVER_80:
         return basicTrainer (this) {
           text "Put 1 Special Energy card attached to 1 of your opponent’s Pokémon in the Lost Zone."
           onPlay {
+            def pcs = opp.all.findAll{it.cards.filterByType(SPECIAL_ENERGY)}.select("Remove special energy from which Pokémon?")
+            pcs.cards.filterByTYpe(SPECIAL_ENERGY).select("Put which special energy in the Lost Zone?").first().moveTo(opp.lostZone)
```

Comment:
Please use the filter parameter instead of `filterByType` here as well.

Reply:
Here, and line 483. Getting into the habit of using the CardList.select()'s `filter` parameter should keep us from causing double prompts like Jirachi from Team Up's.

`moveTo` is only applicable to a CardList, so you can't use `first()` on it before that.

```suggestion
            pcs.cards.select("Put which special energy in the Lost Zone?", cardTypeFilter(SPECIAL_ENERGY)).moveTo(opp.lostZone)
```